drivers/include/net/netdev2.h

```diff
@@ -20,13 +20,19 @@
  *
  * Example call flow:
  *
- * 1. packet arrives -> driver calls netdev2->event_callback with
- *    event==NETDEV_EVENT_ISR (from interrupt routine)
- * 2. event_callback wakes up upper layer thread
- * 3. upper layer calls netdev2->driver.isr()
- * 4. netdev2->driver.isr() calls netdev2->event_callback() with
- *    event==NETDEV_EVENT_RX_COMPLETE
- * 5. netdev2->event_callback() uses netdev2->driver.recv() to fetch packet
+ * 1. packet arrives for device
+ * 2. driver calls @ref netdev2_t::event_callback "netdev->event_callback()"
```

Comment:
Mh but reading the list item again in total it is mostly clear IMHO

> driver calls netdev->event_callback() with `event` := NETDEV2_EVENT_ISR ***(from Interrupt Service Routine)*** and wakes up event handler

Any idea how to make it more clear?